app/components/inputs/SettingsInput.js

```diff
@@ -0,0 +1,59 @@
+import Select from "react-select";
+import { Component } from "react";
+import { autobind } from "core-decorators";
+import { PropTypes } from "prop-types";
+import { injectIntl, intlShape } from "react-intl";
+import "style/Input.less";
+
+@autobind
+class SettingsInput extends Component {
+
+  static propTypes = {
+    intl: intlShape.isRequired,
+    className: PropTypes.string,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: null,
+    };
+  }
+
+  componentWillMount() {
+    const { value } = this.props;
+    if(value) {
+      this.setState({ value: value });
+    }
+  }
+
+  render() {
+    const {
+      className, valueKey, labelKey, multi, clearable, options
+    } = this.props;
+
+    return (
+      <div className={className}>
+        <Select
+          clearable={clearable}
```

Comment:
Duplicated props.  But I think you should just remove clearable and multi as options for the SettingsInput.  I can't see an situation that they would not always both be false for a settings dropdown.  If we did need to add a special case we would just create a specialized one off Input in that case.